docs/android/app_flavors.md

```diff
@@ -0,0 +1,52 @@
+---
+title: "Android Flavors"
+sidebar_label: "Flavors"
+---
+
+:::info
+Only the `:app` and `:automotive` modules are affected by these flavors.
+:::
+
+## Overview
+
+The Android app is built with two flavors: `full` and `minimal`. These flavors allow us to cater to different user preferences. This document explains the differences between the flavors, their features, and the rationale behind their implementation.
+
+## App Flavors
+
+### Full Flavor
+
+The `full` flavor uses the **Google Play Services**, enabling features such as:
+
+- Location tracking
+- Push notifications
+- Communication with Wear OS devices
+
+This flavor is the primary focus of development and is distributed via the Google Play Store.
+
+### Minimal Flavor
+
+The `minimal` flavor is designed for users who prefer or require an app without **Google Play Services**. It is available on F-droid and has the following limitations:
+
+- ❌ No location tracking sensor
+- ❌ No push notifications
```

Comment:
We actually have websocket notifications here which are on by default for `minimal`

https://companion.home-assistant.io/docs/notifications/notification-local#requirements